lib/ohloh_scm/adapters/svn/misc.rb

```diff
@@ -144,10 +144,12 @@ def opt_auth
     #      http://svn.apache.org/repos/asf/maven/plugin-testing/trunk
     #      all have the same root value(https://svn.apache.org/repos/asf)
     def tags
-      tag_strings = `svn log -v #{ base_path}/tags | grep 'tags.\\+(from.\\+:[0-9]\\+)$'`.split(/\n/)
+      tag_strings = `svn ls -v #{ base_path}/tags | tr -s ' ' ' ' | cut -d ' ' -f2,7`.split(/\n/)
       tag_strings.map do |tag_string|
-        tag_string.match(/\/tags\/(.+) \(from .+:(\d+)\)\Z/)[1..2]
-      end
+        commit, tag_name = tag_string.split(/\s/)
+        next if tag_name == './'
```

Comment:
I believe we need to fix `| tr -s ' ' ' ' | cut -d ' ' -f2,7`. Are we purposefully skipping tags ending with more than 1 decimal point?
```
$ svn ls -v https://svn.apache.org/repos/asf/httpd/httpd/tags/
...
 396066 pquerna               Apr 22  2006 2.2.2/
1163057 jim                   Aug 30  2011 2.2.20/
1167190 wrowe                 Sep 09  2011 2.2.21/
1235965 wrowe                 Jan 26  2012 2.2.22/
1375704 wrowe                 Aug 21  2012 2.2.23/
1447538 wrowe                 Feb 19  2013 2.2.24/
1497914 wrowe                 Jun 29  2013 2.2.25/
1541605 jim                   Nov 13  2013 2.2.26/
1577278 wrowe                 Mar 14  2014 2.2.27/
1619760 wrowe                 Aug 22  2014 2.2.28/
1619850 wrowe                 Aug 22  2014 2.2.29/
 426151 wrowe                 Jul 27  2006 2.2.3/
1690340 wrowe                 Jul 11  2015 2.2.30/
1691245 wrowe                 Jul 15  2015 2.2.31/
 493425 wrowe                 Jan 06  2007 2.2.4/
 564746 jim                   Aug 11  2007 2.2.5/
 572801 jim                   Sep 05  2007 2.2.6/
 608857 jim                   Jan 04  2008 2.2.7/
 610841 jim                   Jan 10  2008 2.2.8/
 666274 jim                   Jun 11  2008 2.2.9/
 724095 pquerna               Dec 07  2008 2.3.0/
 730918 pquerna               Jan 03  2009 2.3.1/
1045184 jim                   Dec 13  2010 2.3.10/
1075920 jim                   Mar 01  2011 2.3.11/
1101854 jim                   May 11  2011 2.3.12/
1140732 jim                   Jun 28  2011 2.3.13/
...
```